I do magic/promises-callbacks-js/src/html/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE HTML>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Super Agentes - Home</title>
+    <!--Bootstrap grid-->
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.2/mustache.min.js"></script>
+
+    <script type="text/x-handlebars-template" id="template">
+      {{#companies}}
+        <div class="agent">
+          <p class="agent__name">{{name}}</p>
+          {{#isCertified}}
+          <img src="images/certified.png" alt="ribbon" class="agent__ribbon">
+          {{/isCertified}}
+          <div class="agent__separator"></div>
+          <img class="agent__photo" src="{{image}}">
+          <div class="agent__description">{{description}}</div>
+          <div class="agent__price">Desde: ${{rate}} / {{hours}} horas</div>
+          <div class="agent__button">
+            <div class="agent__button-text">Contratar</div>
+          </div>
+        </div>
+      {{/companies}}
+    </script>
+
+    <script type="text/javascript" src="js/load-data.js"></script>
+  </head>
+
+  <body language=JavaScript onLoad="getAgents('https://api.myjson.com/bins/uptto')">
```

Comment:
Don't load scripts inline. Use another file and load it in this HTML. Remove the `language=JavaScript` tag.

Reply:
Done